nanome_rmsd/rmsd_menu.py

```diff
@@ -18,47 +20,160 @@ def _request_refresh(self):
+                return True
+            else:
+                self.error_message.text_value = ""
+                self.error_message.text_auto_size = True
+                self._plugin.update_content(self.error_message)
+        if(error_type == "select_same"):
+            if self._selected_mobile.complex.index == self._selected_target.complex.index:
+                if from_run == True:
+                    self.error_message.text_auto_size=False
+                    self.error_message.text_size = 0.22
+                    self.error_message.text_value = "Error: Select different complexes"
+                    self.update_score(0,reset=True)
+                    self._plugin.update_content(self.error_message)
+                return True
+            else:
+                self.error_message.text_value = ""
+                self.error_message.text_auto_size = True
+                self._plugin.update_content(self.error_message)
+        if(error_type == "different_size"):
+            if from_run == True:
+                self.error_message.text_auto_size=False
+                self.error_message.text_size = 0.159
+                self.error_message.text_value = "Error: Receptor and target have different sizes"
+                self.update_score(0,reset=True)
+                self._plugin.update_content(self.error_message)
+                return True
+            else:
+                self.error_message.text_value = ""
+                self.error_message.text_auto_size = True
+                self._plugin.update_content(self.error_message)
+        if(error_type == "different_order"):
+            if from_run == True:
+                self.error_message.text_auto_size=False
+                self.error_message.text_size = 0.159
+                self.error_message.text_value = "Error: Receptor and target have different order"
+                self.update_score(0,reset=True)
+                self._plugin.update_content(self.error_message)
+                return True
+            else:
+                self.error_message.text_value = ""
+                self.error_message.text_auto_size = True
+        if(error_type == "zero_size"):
+            if from_run == True:
+                self.error_message.text_auto_size=False
+                self.error_message.text_size = 0.15
+                self.error_message.text_value = "Error: At least one complex has no atom selected"
+                self.update_score(0,reset=True)
+                self._plugin.update_content(self.error_message)
+                return True
+            else:
+                self.error_message.text_value = ""
+                self.error_message.text_auto_size = True
+                self._plugin.update_content(self.error_message)
+
 
     # change the args in the plugin
     def update_args(self,arg,option):
         self._plugin.update_args(arg,option)
 
-    def update_score(self,value):
+    def update_score(self,value,reset=False):
```

Comment:
instead of having a separate reset value, we can just check if "value==None"

Or we should have a reset_score function that is separate from update_score. We generally don't want to have a mandatory useless parameter (like value when we are resetting the score).